templates/repo/issue/choose.tmpl

```diff
@@ -20,17 +20,27 @@
 				</div>
 			</div>
 		{{end}}
-		<div class="ui attached segment">
-			<div class="ui two column grid">
-				<div class="column left aligned">
-					<strong>{{.locale.Tr "repo.issues.choose.blank"}}</strong>
-					<br/>{{.locale.Tr "repo.issues.choose.blank_about"}}
+		{{if .IssueConfig.BlankIssuesEnabled}}
+			<div class="ui attached segment">
+				<div class="ui two column grid">
+					<div class="column left aligned">
+						<strong>{{.locale.Tr "repo.issues.choose.blank"}}</strong>
+						<br/>{{.locale.Tr "repo.issues.choose.blank_about"}}
+					</div>
+					<div class="column right aligned">
+						<a href="{{.RepoLink}}/issues/new?{{if .milestone}}&milestone={{.milestone}}{{end}}{{if $.project}}&project={{$.project}}{{end}}" class="ui green button">{{$.locale.Tr "repo.issues.choose.get_started"}}</a>
+					</div>
 				</div>
-				<div class="column right aligned">
-					<a href="{{.RepoLink}}/issues/new?{{if .milestone}}&milestone={{.milestone}}{{end}}{{if $.project}}&project={{$.project}}{{end}}" class="ui green button">{{$.locale.Tr "repo.issues.choose.get_started"}}</a>
+			</div>
+		{{end}}
+		{{- if .IssueConfigError}}{{/* normal warning flash makes problems here*/}}
+			<div class="ui warning message">
+				<div class="text left">
+					<div>Your issue_config.yaml contains a error:</div>
```

Comment:
Shouldn't that be a part of `options/locale/locale_en-US.ini`?

Reply:
Sorry. I forgot to change it from my tests. I will do it later, when I have time.